cms/apps/media/models.py

```diff
@@ -157,7 +158,12 @@ def __init__(self, **kwargs):
         kwargs.setdefault('on_delete', models.PROTECT)
         super().__init__(**kwargs)
 
-    def formfield(self, **kwargs):
+    def formfield(self, skip_defaults=False, **kwargs):
+        # We skip changing the kwargs if we've already set
+        # the widget in an inheriting class
+        if skip:
```

Comment:
change this to `skip_defaults` (my bad for missing this earlier)

Reply:
I literally just caught this one as you were commenting :p changed now.